controllers/programmes/views/programme.njk

```diff
@@ -44,10 +44,8 @@
                     {% set tabItems = [] %}
                     {% for section in entry.contentSections -%}
                         {% set tabContent -%}
-                            <div class="content-box content-box--frameless">
```

Comment:
These two classes are already added in the `contentTabs` macro, so we were doubling up on spacing, eg:

![broken](https://user-images.githubusercontent.com/394376/47732012-41def480-dc5d-11e8-98ea-8658845fe335.gif)

Reply:
💯